src/org/jitsi/impl/neomedia/transform/srtp/SRTPCipherCTR.java

```diff
@@ -66,63 +63,51 @@ public void init(byte[] key)
     }
 
     /**
-     * Computes the cipher stream for AES CM mode. See section 4.1.1 in RFC3711
-     * for detailed description.
+     * Process (encrypt/decrypt) data from offset for len bytes
+     * iv is modified by this function but you MUST never reuse
+     * an IV so it's ok
      *
-     * @param out byte array holding the output cipher stream
-     * @param length length of the cipher stream to produce, in bytes
-     * @param iv initialization vector used to generate this cipher stream
+     * @param data
+     * @param off
+     * @param len
+     * @param iv initial value of the counter (this value is modified)
      */
```

Comment:
Please document the parameters and include the reference to the RFC. Say something about the cipher stream being computed and used directly.